app/_how-tos/store-a-mistral-api-key-as-a-secret-in-konnect-config-store.md

```diff
@@ -100,27 +106,21 @@ export CONFIG_STORE_ID=config-store-uuid
 
 ## 2. Configure {{site.konnect_short_name}} as your Vault
 
-To enable {{site.konnect_short_name}} as your vault with the [Vault entity](/gateway/entities/vault/) send a `POST` request to the [`/vaults/`](/api/konnect/control-planes-config/v2/#/operations/create-vault) endpoint:
-
+Enable {{site.konnect_short_name}} as your vault with the [Vault entity](/gateway/entities/vault/):
 
-
-<!--vale off-->
-{% control_plane_request %}
-url: /v2/control-planes/$CONTROL_PLANE_ID/core-entities/vaults/
-status_code: 201
-method: POST
-headers:
-    - 'Accept: application/json'
-    - 'Content-Type: application/json'
-    - 'Authorization: Bearer $KONNECT_TOKEN'
-body:
-    config:
-      config_store_id: $CONFIG_STORE_ID
-    description: Storing secrets in Konnect
-    name: konnect
+{% entity_examples %}
+entities:
+  vaults:
+  - name: konnect
     prefix: mysecretvault
-{% endcontrol_plane_request %}
-<!--vale on-->
+    description: Storing secrets in Konnect
+    config:
+      config_store_id: ${config-store-id}
+
+variables:
+  config-store-id:
+    value: $CONFIG_STORE_ID
```

Comment:
On line 98, you need to adjust 
```
export CONFIG_STORE_ID=config-store-uuid
```
to 
```
export DECK_CONFIG_STORE_ID=config-store-uuid
```

So that this command works.